state_cache/Cargo.toml

```diff
@@ -10,7 +10,8 @@ workspace = true
 [dependencies]
 anyhow = { workspace = true }
 im = { workspace = true }
-log = { workspace = true }
+tracing = { workspace = true }
```

Comment:
[**chore**](https://conventionalcomments.org/): Keep dependencies in alphabetical order